src/mn_setup.c

```diff
@@ -3605,6 +3605,7 @@ void MN_CustomSkill(void)
     SetupMenu();
 
     csmenu_skill = default_skill - 1;
+    SelectSkillLevel();
```

Comment:
So all preferences are reset when leaving and returning to menu?

Reply:
Yes, the typical scenario is that the user enters the "custom skill" menu, selects options, and starts the game.
Alternatively, we can maintain the "custom skill" state and save it to the config.

Reply:
> Edit: Maybe too confusing and complicated for this idea.

I want the "custom skill" menu to be simple and not resemble the general options menu with a reset button. For example, the Nugget/Nyan variant looks too complicated to me. However, it's a matter of personal preference. I'm not sure which approach is better.